monorepo-trigger/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: CI for monorepo
+description: Conditionally trigger pipelines within a monorepo based on changes to file paths.
+author: Buildkite
+languages: []
+use_cases: ["Monorepo", "CI"]
+platforms: []
+tools: []
+primary_emojis: [":buildkite:"]
```

Comment:
Open to alternatives for the main emoji.

Perhaps `:git:` + `':buildkite:` makes sense?

Reply:
+1 to git

just being buildkite probs isn't that isntructive